CONTRIBUTING.md

```diff
@@ -30,16 +30,26 @@ backwards-incompatible changes.
 
 - Ensure that GitHub Actions reports no errors.
 
-- Update the version number in `pyproject.toml`.
+- Update the version number in `typing_extensions/pyproject.toml` and in
```

Comment:
nit: for releases I personally find it helpful to split up steps as much as possible and not use `and`

Reply:
I think here it is fine as the step is "update the version number", there are just two places to do it.

A